.gitignore

```diff
@@ -19,3 +19,4 @@ test/version_tmp
 tmp
 *.swp
 .ruby-version
+tags
```

Comment:
@bf4 good tip, I had forgotten about global gitignore.
@joaomdmoura do you prefer to remove this line from .gitignore ? (that's ok for me)